units/commit_queue.go

```diff
@@ -128,7 +128,11 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 		"waiting_secs": time.Since(front.EnqueueTime).Seconds(),
 	})
 
-	conf := j.env.Settings()
+	conf, err := evergreen.GetConfig()
```

Comment:
this allows us to turn on/off the new batching behavior without having to bounce the apps